### DISCUSSION TITLE
Dreambooth - `Error: 'prompt'`

### DISCUSSION THREAD

**Discussion by flamed0g** [huggingface/autotrain-advanced#652] [Q&A] [answered]
Hi there,

I am trying to train using Dreambooth on Google Colab but for some reason I am getting the following error:
```Training is starting... Please wait!
An error occurred while starting training!
Error: 'prompt'
```

As shown in the screenshot down below:

![image](https://github.com/huggingface/autotrain-advanced/assets/53303556/1d4e102e-f3d9-4540-a6ee-6438ee75feed)

I am probably missing something obvious but I can't seem to figure it out..

**Comment by abhishekkrthakur** [comment, marked as answer]
just pushed a fix. thanks for reporting this. 
please re-initialize the colab and it should work. make sure you are on version 0.7.103 or above.
